### DISCUSSION TITLE
Demo Account

### DISCUSSION THREAD

**Discussion by alanrenouf** [jobvk/Home-Assistant-Windcentrale#100] [General]
I have the integration installed but was wondering if there was a demo account that anyone knew of that could be used to show what this data looks like?  Im working on a project where showing this data would be fantastic.  Thanks

**Comment by alanrenouf** [comment]
Or maybe there is a mock API I can use?

**Reply by jobvk** [reply]
No, there is no demo account or mock API because the data is person-specific and private data can be retrieved. You would have to buy wind shares for getting access to data.
